pkg/virt-api/webhooks/validating-webhook/admitters/vmi-create-admitter.go

```diff
@@ -1629,6 +1629,26 @@ func validateDisks(field *k8sfield.Path, disks []v1.Disk) []metav1.StatusCause {
 			})
 		}
 
+		ioModeOK := false
+		switch disk.IO {
+		case "":
+			ioModeOK = true
+		case "native":
+			ioModeOK = true
+		case "default":
```

Comment:
How will this be converted?

Reply:
`default` is a valid value, it tells QEMU to use whatever is configured as kernel's default IO mode. Usually it's async io.

Reply:
what if we don't provide value at all? libvirt defaults to `threads` right?